libs/util/shared/src/lib/utils/strip-html.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { stripHtml } from './strip-html'
+
+describe('strip HTML', () => {
+  describe('when HTML tags', () => {
+    it('removes html tags', () => {
+      const html = '<b>hello</b>'
+      expect(stripHtml(html)).toBe('hello')
+    })
+  })
+  describe('when no HTML tags', () => {
+    it('return same string', () => {
+      const html = 'hello'
+      expect(stripHtml(html)).toBe('hello')
+    })
+  })
```

Comment:
It'd be interesting to add a test to make sure that any js in the html is _not_ executed, something like a console log (but this is just nice to have)